api/opensource/mfx_dispatch/include/intel_api_factory.h

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2018 Intel Corporation
```

Comment:
Please, align location with Linux dispatcher:
1. Drop "opensource" folder: the whole this repository is in the open source, the name is misleading
2. Locate windows distpatcher in the dedicated folder:  api/mfx_dispatch/windows/. Creating again the dispatcher in the old location would give confusions.